### PR TITLE
Build/Run Scripts from Parkstation

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

Adds several scripts to make building and running the projects easier outside of an IDE.

### CODE REVIEW

**Review by VMSolidus — APPROVED** [review]
I have just finished testing the entire .bat side of this PR, but since I don't have any way of running shell scripts on my monster rig(and I'm not going to go install Linux or a VM just to test this PR), I'm giving my approval for that half of the PR. Psprite wants to verify that the Shell half of these scripts works as advertised, they can be my guest. Otherwise I'll leave it up to them to merge this. This has my approval so far.

**Comment by DEATHB4DEFEAT** [comment]
I tested the batch scripts in cmd and the shell scripts in git bash, Linux users may need to add the executable permission themselves though.

**Review by stellar-novas — COMMENTED** [review]
Linux script permissions are wrong, the owner at least should have execute, so 744/rwxr--r--

File: Scripts/bat/!README.txt
Comment (DangerRevolution) [inline Scripts/bat/!README.txt:8]: Mapping should use the tools build to stop mapinits

**Comment by stellar-novas** [comment]
As for the scripts themselves, yeah good though it might be nice to have a Tools one, as Danger said.

**Comment by DEATHB4DEFEAT** [comment]
I'll set the permissions and add a Tools build when I get time

**Comment by DEATHB4DEFEAT** [comment]
I have returned home.

**Review by DangerRevolution — APPROVED** [review]
i'd merge but looks like you're still working on it :)

**Pull request merged**
